src/traces/box/attributes.js

```diff
@@ -378,6 +380,15 @@ module.exports = {
         ].join(' ')
     },
 
+    whiskerdisable: {
+        valType: 'boolean',
+        dflt: false,
```

Comment:
Let's avoid the double negative: call the attribute `showwhiskers`. And assuming that there isn't normally a purpose for whiskers in this new "mean ±N sd" mode, we can make the default `true` when we're showing quartiles and `false` for sd. To do this, you calculate the default in `defaults.js` after coercing the attribute(s) it depends on, like we do with `boxmeanDflt`, and describe this logic in words in the below `description`.

What happens if you DO show whiskers with mean/sd boxes? Are they still at the upper/lower fence positions even though those don't mean much without the quartiles?